test/elixir_google_scraper/users_test.exs

```diff
@@ -48,6 +48,20 @@ defmodule ElixirGoogleScraper.UsersTest do
     end
   end
 
+  describe "authenticate/2" do
+    test "authenticates successfully when the credentialsare valid" do
```

Comment:
Maybe change the description to something like

```
returns {:ok, user} when the credentials are valid
```
Because when reading `authenticates successfully`, we might not know what does it do 🤔 💡